@xen-orchestra/lite/src/components/PanelHeader.tsx

```diff
@@ -37,8 +45,8 @@ interface Props extends React.ButtonHTMLAttributes<HTMLButtonElement> {
 const PanelHeader = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
   {},
   ({ actions = [], titleProps = {}, buttonGroupProps = {}, children = null }) => (
-    <Stack direction='row' alignItems='start'>
-      <Typography variant='h1' sx={DEFAULT_TITLE_STYLE} {...titleProps}>
+    <Stack alignItems='start' direction='row' sx={DEFAULT_STACK_STYLE}>
```

Comment:
Is it intended to put every thing on start , or do you want the buttongroup on the end of the line ( align=space-between ) ?